src/js/views/patients/worklist/filters/date-filter_state.js

```diff
@@ -22,4 +23,63 @@ export default Backbone.Model.extend({
       selectedMonth: null,
     });
   },
+  incrementBackward() {
+    if (this.get('selectedDate')) {
+      this.setDate(dayjs(this.get('selectedDate')).subtract(1, 'day'));
+      return;
+    }
+
+    if (this.get('selectedMonth')) {
+      this.setMonth(dayjs(this.get('selectedMonth')).subtract(1, 'month').startOf('month'));
+      return;
+    }
+
+    const relativeDate = this.get('relativeDate');
+
+    if (relativeDate === 'today') {
+      this.setDate(dayjs().subtract(1, 'day'));
+      return;
+    }
+
+    if (relativeDate === 'yesterday') {
+      this.setDate(dayjs().subtract(2, 'days'));
+      return;
+    }
+
+    if (this._isCurrentMonth()) {
```

Comment:
This and the same conditional below leave branches untested: the `!this._isCurrentMonth` case.

The solution is to just make the `setMonth` call the default fallback, but I was hesitant to do that in case there was some case I was missing. I've looked this over a dozen times and haven't come up with a scenario where `this._isCurrentMonth` wouldn't be true at this point in the method. So it may be safe to remove the check.

But I'll take a look on Monday with fresh eyes and verify later. Or maybe there's a better way to structure this so we don't have an untested branch.

Reply:
yep I think it can be removed